recipe_scrapers/wellplated.py

```diff
@@ -0,0 +1,42 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+
+
+class WellPlated(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "wellplated.com"
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def instructions(self):
+        instructions = self.schema.instructions()
+        if isinstance(instructions, list):
+            instructions = "\n".join(instructions)
```

Comment:
Out of curiosity: did you find pages on this website where the `instructions` was not a list?

Reply:
After looking back through there doesn't appear to be any. I likely carried that over from another scraper by mistake. I've removed lines 31 & 32